docs/superbench-config.mdx

```diff
@@ -208,7 +214,7 @@ ${benchmark_name}:
 #### Model-Benchmark
 
 ```yaml
-${name}_models:
+model-benchmarks:${annotation}:
```

Comment:
Do we need to update the config files to align with the docs?

Reply:
it's backward competitive, current config files still work
we can update once when release next version